app/indexers/concerns/iiif_print/file_set_indexer.rb

```diff
@@ -35,7 +35,7 @@ def find_checksum(object)
       return unless file
 
       digest ||= if file.is_a?(Hyrax::FileMetadata)
-                   file.checksum
+                   Array.wrap(file.checksum).first
```

Comment:
Checksum doesn't seem to work, but it seemed to be an array sometimes and a string others, so I chose to use Array.wrap for consistency.